src/test/java/dev/openfga/sdk/api/OpenFgaApiTest.java

```diff
@@ -148,7 +159,7 @@ public void listStores_500() throws Exception {
                         .get());
 
         // Then
-        mockHttpClient.verify().get("https://localhost/stores").called(1);
+        mockHttpClient.verify().get("https://localhost/stores").called(1 + DEFAULT_MAX_RETRIES);
         ApiException exception = assertInstanceOf(ApiException.class, execException.getCause());
```

Comment:
Do we want to expect a more specific class such as FgaApiInternalError instead of the generic ApiException?
Similar for other errors

Reply:
Good call, I will update these

Reply:
[ecbfe49](https://github.com/openfga/sdk-generator/pull/217/commits/ecbfe4900bfebcca4e5ae543f6e2214926414308)